azure-pipelines/publishing/github-release.yml

```diff
@@ -9,96 +9,95 @@ pr:
+        config: release
+    - template: ../build/templates/dopapt-docker-steps.yml
+      parameters:
+        targetOsArch: debian10-arm32
+        dockerImageName: 'jimsonmsft/debian10-arm32:latest'
+        config: release
+    - template: ../build/templates/dopapt-docker-steps.yml
+      parameters:
+        targetOsArch: debian9-arm32
+        dockerImageName: 'jimsonmsft/debian9-arm32:latest'
+        config: release
+    - template: ../build/templates/dopapt-docker-steps.yml
+      parameters:
+        targetOsArch: ubuntu1804-arm64
+        dockerImageName: 'jimsonmsft/ubuntu18.04-arm64:latest'
+        config: release
 
-  - task: DownloadBuildArtifacts@0
-    inputs:
-      buildType: 'specific'
-      project: '7cdf52bf-a6f7-436c-b12f-3f063ef389c4'
-      pipeline: '25'
-      buildVersionToDownload: 'latest'
-      downloadType: 'specific'
-      downloadPath: '/tmp/do_publishing'
-    displayName: 'Get ms-do-agent arm artifacts'
-
-  - task: DownloadBuildArtifacts@0
-    inputs:
-      buildType: 'specific'
-      project: '7cdf52bf-a6f7-436c-b12f-3f063ef389c4'
-      pipeline: '23'
-      buildVersionToDownload: 'latest'
-      downloadType: 'specific'
-      downloadPath: '/tmp/do_publishing'
-    displayName: 'Get ms-do-agent x64 artifacts'
-
-  - task: DownloadBuildArtifacts@0
-    inputs:
-      buildType: 'specific'
-      project: '7cdf52bf-a6f7-436c-b12f-3f063ef389c4'
-      pipeline: '31'
-      buildVersionToDownload: 'latest'
-      downloadType: 'specific'
-      downloadPath: '/tmp/do_publishing'
-    displayName: 'Get ms-do-pluginapt arm artifacts'
-
-  - task: DownloadBuildArtifacts@0
-    inputs:
-      buildType: 'specific'
-      project: '7cdf52bf-a6f7-436c-b12f-3f063ef389c4'
-      pipeline: '29'
-      buildVersionToDownload: 'latest'
-      downloadType: 'specific'
-      downloadPath: '/tmp/do_publishing'
-    displayName: 'Get ms-do-pluginapt x64 artifacts'
-
-  - task: ArchiveFiles@2
-    inputs:
-      rootFolderOrFile: '/tmp/do_publishing'
-      includeRootFolder: true
-      archiveType: 'tar'
-      archiveFile: '$(Build.ArtifactStagingDirectory)/artifacts.tar.gz'
-      replaceExistingArchive: true
-
-  - task: GitHubRelease@1
-    inputs:
-      gitHubConnection: 'client2'
-      repositoryName: 'microsoft/do-client'
-      action: 'create'
-      target: '$(Build.SourceVersion)'
-      tagSource: 'userSpecifiedTag'
-      tag: 'ms-do-client-$(release.version)'
-      title: '$(release.title) $(release.version)'
-      changeLogCompareToRelease: 'lastNonDraftReleaseByTag'
-      changeLogCompareToReleaseTag: 'ms-do-client-$(release.previous_version)'
-      changeLogType: 'commitBased'
\ No newline at end of file
+- stage: release
+  jobs:
+  - job: github_release
+    steps:
+    - task: GitHubRelease@1
+      inputs:
+        gitHubConnection: 'client2'
+        repositoryName: 'microsoft/do-client'
+        action: 'create'
+        assets: '$(Build.ArtifactStagingDirectory)/**/*-release/*'
```

Comment:
>* [](start = 64, length = 1)

Consider specifying *.deb. That is the only asset we intend to publish, yes?

Reply:
Correct

---
In reply to: [585981245](https://github.com/microsoft/do-client/pull/7#discussion_r585981245) [](ancestors = 585981245)